Makefile

```diff
@@ -40,7 +40,7 @@ INCLUDE_PATH = include
 BUILD_PATH = out
 
 # Tool flags
-CFLAGS = -nostdlib --freestanding -fno-builtin -g -I$(INCLUDE_PATH)/
+CFLAGS = -nostdlib --freestanding -fno-builtin -g -I $(INCLUDE_PATH)/
```

Comment:
This isn't actually a bug. GCC won't care whether you have a space after -I or not.

In fact, until today I thought you were supposed not to, because I have always see this flag without any spaces; but, because this compiles, I guess it's actually legal to have a space there.